conformance/1.7/README.md

```diff
@@ -0,0 +1,27 @@
+# Kubeflow conformance program (WIP)
+
+Before running the conformance tests, you need to configure kubectl default context to point to the k8s cluster that is hosting Kubeflow.
+
+TODO: Make the kubeflow namespace configurable.
```

Comment:
NIT: move this out of `README.md`. `Makefile`, maybe?

Reply:
Removed. The KF service namespace was made configurable.